### ISSUE TITLE
Empty Screen running on Ubuntu 22.04 Raspberry pi 4

### ISSUE THREAD

**Issue by TheAfr0man** [flutter/flutter#132043]
Hi, I'm trying to run the Hello World example on Ubuntu 22.04 running on Raspberry pi 4. But I'm getting an empty white screen as shown in the screenshot. (PS: I'm new to flutter)

OS Image : Ubuntu 22.04.2 LTS 5.15.0-1034-raspi

![Screenshot from 2023-08-07 16-07-42](https://github.com/flutter/flutter/assets/76805673/fe3fea42-bffa-4a5a-919f-8083a2cf3cda)

Also tried (https://stackoverflow.com/questions/62809877/gdk-critical-exceptions-on-a-flutter-desktop-app-linux) disabling wayland and restarting gnome as shown in the accepted answer. But it didnt work for me

Running the app as shown below

```
drongopi1@drongopi1:~/snap/flutter/common/flutter/examples/hello_world$ flutter run -d linux
Launching lib/main.dart on Linux in debug mode...
Building Linux application...                                           

(hello_world:5130): Gdk-CRITICAL **: 16:05:51.404: gdk_window_get_state: assertion 'GDK_IS_WINDOW (window)' failed
Syncing files to device Linux...                                   388ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h List all available interactive commands.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).

A Dart VM Service on Linux is available at: http://127.0.0.1:40529/ilbmD5GIboo=/
The Flutter DevTools debugger and profiler on Linux is available at:
http://127.0.0.1:9100?uri=http://127.0.0.1:40529/ilbmD5GIboo=/

```

Flutter doctor output
```
drongopi1@drongopi1:~$ flutter doctor -v
[✓] Flutter (Channel master, 3.13.0-17.0.pre.80, on Ubuntu 22.04.2 LTS
    5.15.0-1034-raspi, locale en_IN)
    • Flutter version 3.13.0-17.0.pre.80 on channel master at
      /home/drongopi1/snap/flutter/common/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision c11ac7be9a (57 minutes ago), 2023-08-07 05:43:32 -0400
    • Engine revision b10891e0d8
    • Dart version 3.2.0 (build 3.2.0-41.0.dev)
    • DevTools version 2.26.1

[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from:
      https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK
      components.
      (or visit https://flutter.dev/docs/get-started/install/linux#android-setup
      for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.


[✗] Chrome - develop for the web (Cannot find Chrome executable at
    google-chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[✓] Linux toolchain - develop for Linux desktop
    • clang version 10.0.0-4ubuntu1
    • cmake version 3.16.3
    • ninja version 1.10.0
    • pkg-config version 0.29.1

[!] Android Studio (not installed)
    • Android Studio not found; download from
      https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/linux#android-setup
      for detailed instructions).

[✓] Connected device (1 available)
    • Linux (desktop) • linux • linux-arm64 • Ubuntu 22.04.2 LTS
      5.15.0-1034-raspi

[✓] Network resources
    • All expected network resources are available.

! Doctor found issues in 3 categories.

```

**Comment by darshankawar** [comment]
@TheAfr0man 
Can you run `flutter run -d linux -v` and share the entire verbose log ? If it the log is huge, put that in a text file and share it here.

**Comment by TheAfr0man** [comment]
@darshankawar the log is as shown below
```
drongopi1@drongopi1:~/snap/flutter/common/flutter/examples/hello_world$ flutter run -d linux -v
[  +11 ms] executing: [/home/drongopi1/snap/flutter/common/flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +35 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms] f468f3366c26a5092eb964a230ce7892fda8f2f8
[   +1 ms] executing: [/home/drongopi1/snap/flutter/common/flutter/] git tag --points-at f468f3366c26a5092eb964a230ce7892fda8f2f8
[  +66 ms] Exit code 0 from: git tag --points-at f468f3366c26a5092eb964a230ce7892fda8f2f8
[   +1 ms] 3.10.6
[   +9 ms] executing: [/home/drongopi1/snap/flutter/common/flutter/] git rev-parse --abbrev-ref HEAD
[  +25 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms] stable
[  +61 ms] Unable to locate an Android SDK.
[  +12 ms] executing: [/home/drongopi1/snap/flutter/common/flutter/] git rev-parse --abbrev-ref --symbolic @{upstream}
[  +26 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{upstream}
[   +1 ms] origin/stable
[        ] executing: [/home/drongopi1/snap/flutter/common/flutter/] git ls-remote --get-url origin
[  +30 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms] https://github.com/flutter/flutter.git
[   +4 ms] executing: uname -m
[  +26 ms] Exit code 0 from: uname -m
[   +1 ms] aarch64
[ +285 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +6 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[  +19 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +339 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +6 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[  +14 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[  +10 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +520 ms] Skipping pub get: version match.
[ +952 ms] Generating /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java
[ +360 ms] Initializing file store
[  +41 ms] Skipping target: gen_localizations
[  +31 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents:
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/.dart_tool/package_config_subset}
[ +192 ms] gen_dart_plugin_registrant: Complete
[   +3 ms] Skipping target: _composite
[   +6 ms] complete
[  +31 ms] Launching lib/main.dart on Linux in debug mode...
[  +17 ms] /home/drongopi1/snap/flutter/common/flutter/bin/cache/dart-sdk/bin/dart --disable-dart-dev
/home/drongopi1/snap/flutter/common/flutter/bin/cache/dart-sdk/bin/snapshots/frontend_server.dart.snapshot --sdk-root
/home/drongopi1/snap/flutter/common/flutter/bin/cache/artifacts/engine/common/flutter_patched_sdk/ --incremental --target=flutter --experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true
-DFLUTTER_WEB_CANVASKIT_URL=https://www.gstatic.com/flutter-canvaskit/cdbeda788a293fa29665dc3fa3d6e63bd221cb0d/ --output-dill /tmp/flutter_tools.XMLPRO/flutter_tool.VOODOT/app.dill --packages
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/.dart_tool/package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --filesystem-scheme
org-dartlang-root --initialize-from-dill build/ade00fc64fa695e0daed02c1105f6d67.cache.dill.track.dill --verbosity=error --enable-experiment=alternative-invalidation-strategy
[ +120 ms] Building Linux application...
[  +68 ms] <- compile package:hello_world/main.dart
[  +13 ms] executing: [build/linux/arm64/debug/] cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug -DFLUTTER_TARGET_PLATFORM=linux-arm64 /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux
[ +381 ms] -- Configuring done
[ +146 ms] -- Generating done
[   +4 ms] -- Build files have been written to: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug
[  +20 ms] executing: ninja -C build/linux/arm64/debug install
[  +42 ms] ninja: Entering directory `build/linux/arm64/debug'
[+11415 ms] [1/5] Generating /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/libflutter_linux_gtk.so,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_basic_message_channel.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_binary_codec.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_binary_messenger.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_dart_project.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_engine.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_json_message_codec.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_json_method_codec.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_message_codec.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_method_call.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_method_channel.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_method_codec.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_method_response.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_plugin_registrar.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_plugin_registry.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_standard_message_codec.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_standard_method_codec.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_string_codec.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_value.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/fl_view.h,
/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/linux/flutter/ephemeral/flutter_linux/flutter_linux.h, _phony_
[  +19 ms] [   +9 ms] executing: [/home/drongopi1/snap/flutter/common/flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms] [  +35 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms] [   +1 ms] f468f3366c26a5092eb964a230ce7892fda8f2f8
[   +1 ms] [   +1 ms] executing: [/home/drongopi1/snap/flutter/common/flutter/] git tag --points-at f468f3366c26a5092eb964a230ce7892fda8f2f8
[   +1 ms] [  +57 ms] Exit code 0 from: git tag --points-at f468f3366c26a5092eb964a230ce7892fda8f2f8
[   +8 ms] [   +1 ms] 3.10.6
[   +3 ms] [  +10 ms] executing: [/home/drongopi1/snap/flutter/common/flutter/] git rev-parse --abbrev-ref HEAD
[   +3 ms] [  +20 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[  +15 ms] [   +1 ms] stable
[        ] [  +39 ms] Unable to locate an Android SDK.
[        ] [  +13 ms] executing: [/home/drongopi1/snap/flutter/common/flutter/] git rev-parse --abbrev-ref --symbolic @{upstream}
[        ] [  +20 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{upstream}
[  +20 ms] [        ] origin/stable
[   +1 ms] [        ] executing: [/home/drongopi1/snap/flutter/common/flutter/] git ls-remote --get-url origin
[        ] [  +19 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] [        ] https://github.com/flutter/flutter.git
[  +12 ms] [   +3 ms] executing: uname -m
[   +1 ms] [  +15 ms] Exit code 0 from: uname -m
[        ] [   +1 ms] aarch64
[   +5 ms] [ +101 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms] [   +1 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +7 ms] [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ] [   +7 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[  +11 ms] [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ] [ +797 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ] [   +1 ms] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ] [   +1 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] [   +1 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms] [   +1 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms] [   +1 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +2 ms] [   +1 ms] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[   +1 ms] [   +2 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] [   +1 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] [  +14 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] [   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ] [   +1 ms] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [   +1 ms] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [   +1 ms] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ] [   +1 ms] Artifact Instance of 'PubDependencies' is not required, skipping update.
[        ] [ +161 ms] Initializing file store
[        ] [  +27 ms] Done initializing file store
[        ] [ +138 ms] Skipping target: gen_localizations
[        ] [  +33 ms] Skipping target: gen_dart_plugin_registrant
[        ] [+1556 ms] Skipping target: unpack_linux
[   +1 ms] [ +814 ms] Skipping target: kernel_snapshot
[   +1 ms] [ +481 ms] Skipping target: debug_bundle_linux-arm64_assets
[   +1 ms] [   +3 ms] Persisting file store
[   +1 ms] [  +27 ms] Done persisting file store
[   +1 ms] [  +16 ms] build succeeded.
[        ] [  +32 ms] "flutter assemble" took 4,166ms.
[        ] [ +316 ms] ensureAnalyticsSent: 269ms
[        ] [        ] Running 0 shutdown hooks
[   +1 ms] [        ] Shutdown hooks complete
[   +1 ms] [   +7 ms] exiting with code 0
[+2291 ms] [2/5] Building CXX object CMakeFiles/hello_world.dir/flutter/generated_plugin_registrant.cc.o
[ +157 ms] [3/5] Building CXX object CMakeFiles/hello_world.dir/my_application.cc.o
[ +516 ms] [4/5] Linking CXX executable intermediates_do_not_run/hello_world
[        ] [4/5] Install the project...
[  +58 ms] -- Install configuration: "Debug"
[  +30 ms] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/hello_world
[   +3 ms] -- Set runtime path of "/home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/hello_world" to "$ORIGIN/lib"
[   +1 ms] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/data/icudtl.dat
[   +8 ms] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/lib/libflutter_linux_gtk.so
[ +266 ms] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/data/flutter_assets
[        ] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/data/flutter_assets/FontManifest.json
[   +1 ms] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/data/flutter_assets/version.json
[   +1 ms] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/data/flutter_assets/shaders
[        ] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/data/flutter_assets/shaders/ink_sparkle.frag
[   +1 ms] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/data/flutter_assets/AssetManifest.json
[   +1 ms] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/data/flutter_assets/AssetManifest.bin
[   +1 ms] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/data/flutter_assets/kernel_blob.bin
[ +152 ms] -- Installing: /home/drongopi1/snap/flutter/common/flutter/examples/hello_world/build/linux/arm64/debug/bundle/data/flutter_assets/NOTICES.Z
[  +51 ms] Building Linux application... (completed in 15.8s)
[+1349 ms] VM Service URL on device: http://127.0.0.1:35249/weuXZhis7ao=/
[  +27 ms] Caching compiled dill
[ +456 ms] Connecting to service protocol: http://127.0.0.1:35249/weuXZhis7ao=/
[ +638 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at http://127.0.0.1:35249/weuXZhis7ao=/.
[ +352 ms] DDS is listening at http://127.0.0.1:43519/an53B-Jez6o=/.
[ +348 ms] Successfully connected to service protocol: http://127.0.0.1:35249/weuXZhis7ao=/
[ +178 ms] DevFS: Creating new filesystem on the device (null)
[ +123 ms] DevFS: Created new filesystem on the device (file:///tmp/hello_worldKHUVZY/hello_world/)
[  +17 ms] Updating assets
[ +489 ms] Syncing files to device Linux...
[   +8 ms] Compiling dart to kernel with 0 updated files
[   +1 ms] Processing bundle.
[   +5 ms] <- recompile package:hello_world/main.dart b1d37c06-ace7-46e1-a963-cf5fe746f90e
[   +1 ms] <- b1d37c06-ace7-46e1-a963-cf5fe746f90e
[   +8 ms] Bundle processing done.
[ +293 ms] Updating files.
[        ] DevFS: Sync finished
[   +5 ms] Syncing files to device Linux... (completed in 321ms)
[   +2 ms] Synced 0.0MB.
[   +6 ms] <- accept
[  +11 ms] Connected to _flutterView/0x9bb6f50.
[  +15 ms] Flutter run key commands.
[   +7 ms] r Hot reload. 🔥🔥🔥
[   +4 ms] R Hot restart.
[   +1 ms] h List all available interactive commands.
[   +5 ms] d Detach (terminate "flutter run" but leave application running).
[   +1 ms] c Clear the screen
[   +1 ms] q Quit (terminate the application on the device).
[   +2 ms] A Dart VM Service on Linux is available at: http://127.0.0.1:43519/an53B-Jez6o=/
[ +517 ms] The Flutter DevTools debugger and profiler on Linux is available at: http://127.0.0.1:9100?uri=http://127.0.0.1:43519/an53B-Jez6o=/
[+22325 ms] Application finished.


```

**Comment by darshankawar** [comment]
Thanks for the verbose log, which doesn't show any errors. Does the empty screen come up everytime you try to run ? 
Maybe, also take a look at https://bruce-abernethy.medium.com/flutter-on-raspberry-pi-fall-2022-ac0766839b92 and see if it helps or not.
